lib/internal/process/esm_loader.js

```diff
@@ -58,41 +59,61 @@ async function initializeLoader() {
   const { getOptionValue } = require('internal/options');
   const customLoaders = getOptionValue('--experimental-loader');
   const preloadModules = getOptionValue('--import');
-  const loaders = await loadModulesInIsolation(customLoaders);
+
+  let cwd;
+  try {
+    cwd = process.cwd() + '/';
+  } catch {
+    cwd = '/';
```

Comment:
This was `cwd = 'file:///` in the original version; perhaps this might have to do with some of the tests I see failing on certain platforms? Those failures don’t look like flakes.